From f2f255eb3fceb8a1edab687ed5d3e846792df099 Mon Sep 17 00:00:00 2001
From: jinghua <jinghua@marvell.com>
Date: Thu, 23 Jun 2016 00:38:41 +0800
Subject: [PATCH 1232/1240] doc: a3700: remove wtp tool install description
 from build.txt

Remove WTP tool install description from build.txt, since
- wtp tool itself has been moved to ATF folder.
- wtp tool used to build whole u-boot image which includes
  tim header, spl.bin and u-boot.bin, but now nothing is built
  other than u-boot.bin in u-boot. So wtp tool is not needed here.
- The description of wtp tool installation has been
  moved to ATF build.txt document.

Change-Id: Ibcc7deeaecf5e9b979d5ed41bbec86e6182f871c
Signed-off-by: jinghua <jinghua@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30661
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Reviewed-by: Gina Tadmore <gina@marvell.com>
---
 doc/mvebu/build.txt | 35 +----------------------------------
 1 file changed, 1 insertion(+), 34 deletions(-)

diff --git a/doc/mvebu/build.txt b/doc/mvebu/build.txt
index cc4e5ad..77f318b 100644
--- a/doc/mvebu/build.txt
+++ b/doc/mvebu/build.txt
@@ -36,8 +36,7 @@ Build Instruction:
 	For the Armada-8040-RZ board use "make mvebu_armada70x0_rz_defconfig"
 	For the Armada-70x0 board use "make mvebu_armada70x0_defconfig"
 	For the Armada-3700 board use "make mvebu_armada3700_defconfig"
-7. Build the U-Boot (Armada-3700 builds required WTP tools installation -
-   Refer to the section "Tools installation for Armada-3700 builds"):
+7. Build the U-Boot:
 	# make
 	The U-Boot compilation generate:
 	u-boot.bin - U-Boot image
@@ -121,35 +120,3 @@ Boot device selection note for Armada-3700 builds:
 	to swap polarity on SATA RX and/or TX lines.
 
 
-Tools installation for Armada-3700 builds:
-------------------------------------------
-1. Extract the tools/wtp/WTPTP_TOOLS_x_x_xx.zip into a temporary directory
-
-   # cd tools/wtp
-   # 7z x WTPTP_TOOLS_x_x_xx.zip
-
-2. If required, add executable permissions to files in the extracted
-   WTPTP_TOOLS_x_x_xx/Linux_Tools folder:
-
-   # sudo chmod a+x WTPTP_TOOLS_x_x_xx/Linux_Tools/*.exe
-
-3. Copy executable files from WTPTP_TOOLS_x_x_xx/Linux_Tools
-   to location pointed to $PATH environment variable.
-   For instance:
-
-   # sudo cp WTPTP_TOOLS_x_x_xx/Linux_Tools/*.exe /usr/local/bin
-
-4. Make sure the "ntbb_linux.exe" can be executed from any
-   location and the U-Boot "make" passes without error.
-
-5. Install a cross GNU ARM tool chain for building the WTMI binary.
-   Any cross GNU ARM tool chain able to build ARM Cortex M3 binaries
-   is suitable.
-   On Debian/Uboot hosts the default GNU ARM tool chain can be installed
-   using the following command:
-   # sudo apt-get install gcc-arm-linux-gnueabi
-   If required, the default tool chain prefix "arm-linux-gnueabi-" can be
-   overwritten using environment variable CROSS_CM3.
-   Example for BASH shell:
-   # export CROSS_CM3=/opt/arm-cross/bin/arm-linux-gnueabi
-
-- 
1.9.1

